site/content/concepts/tenancy.md

```diff
@@ -132,18 +128,18 @@ Here is a summary of how this is implemented:
 - the registration of devices are strictly scoped to a tenant
 - the credentials of devices are strictly scoped to a tenant
 - protocol adapters can be enabled/disabled for a tenant 
-- the upstream data flow is isolated for every tenant
+- the downstream data flow is isolated for every tenant
 - *Business applications* need to authenticate to the AMQP 1.0 network and are by that mechanism scoped to their tenant
  
 ## Notes on a production ready setup
 
 To be flexible for the different needs of production setups, Hono tries to make as few assumptions about the combination
 of the different APIs as possible.
-This means e.g. that there is no connection between the example implementations of the APIs:
+This means e.g. that the Device Registry does not enforce referential integrity of the APIs:
 
 - devices can be created for a tenant that is not existing (yet)
 - credentials can be created for a tenant and/or a device that is not existing (yet)
 - tenants can be deleted and leave their scoped devices and credentials still in the configuration (which may not be usable
   anymore, since the tenant is missing)
 
-These are points that production setups may want to implement differently.
+These points production setups may want to implement differently.
```

Comment:
You seem to have garbled up the words of this sentence ;-)

Reply:
If that did not make the sentence more understandable, I just roll it back ;-)